src/components/Form/Fields/styles.scss

```diff
@@ -22,7 +22,7 @@
     }
   }
 
-  // &-error {
-  //   color: $error-colour;
-  // }
+  &-dropdown {
```

Comment:
I know that we haven't got specific designs for what the model should look like, but can you try to get the dropdown with a consistent style to the input box. It should just be a matter of increasing its height and having a grey outline on it.